examples/react/nextjs-swr/hooks/use-query.ts

```diff
@@ -20,7 +20,7 @@ export function useGraphQL<TResult, TVariables>(
       document.definitions.find(isOperationDefinition)?.name,
       variables,
     ] as const,
-    async (_key: string, variables: any) =>
+    async ([_key, variables]: any) =>
```

Comment:
Looks like the current version of swr keeps the array as-is: https://swr.vercel.app/docs/arguments#multiple-arguments